src/components/views/ViewCampaign.jsx

```diff
@@ -159,6 +159,11 @@ class ViewCampaign extends Component {
                     currentUser={currentUser}
                   />
                 )}
+                {campaign.myCommunityUrl && (
```

Comment:
Instead of using the "private" atribute of the model (ueah ES6 does not have private atributes, can you use the setter? https://github.com/Giveth/giveth-dapp/blob/develop/src/models/Campaign.js#L113

Just name replace `myCommunityUrl` with `communityUrl`